engine/php7-debian.go

```diff
@@ -7,6 +7,6 @@
 
 package engine
 
-// #cgo CFLAGS: -I/usr/include/php/20151012 -Iinclude/php7 -Isrc/php7
+// #cgo CFLAGS: -I/usr/include/php/20151012 -I/usr/include/php/20151012/main -I/usr/include/php/20151012/Zend -I/usr/include/php/20151012/TSRM -Iinclude/php7 -Isrc/php7
```

Comment:
Could you break these in two lines, as in [php5.go](https://github.com/deuill/go-php/blob/daf6987b14a969e3935127753e0a4cec6c3fd768/engine/php5.go#L5)? Otherwise looks great.